src/main/java/org/mskcc/cbio/oncokb/service/impl/TokenServiceImpl.java

```diff
@@ -106,10 +118,44 @@ public void increaseTokenUsage(Long id, int increment) {
         return tokenRepository.findAllExpiresBeforeDate(date);
     }
 
+    @Override
+    public void expireToken(Token token) {
+        token.setExpiration(Instant.now());
+        save(token);
```

Comment:
Moved the `expireToken` method from `TokenProvider` to `TokenService` to avoid the circular dependencies.